doc/lua_api.txt

```diff
@@ -7126,10 +7145,10 @@ object you are working with still exists.
     * in third person view (max. values `{x=-10/10,y=-10,15,z=-5/5}`)
 * `get_eye_offset()`: returns first and third person offsets.
 * `send_mapblock(blockpos)`:
-    * Sends a server-side loaded mapblock to the player.
-    * Returns `false` if failed.
+    * Sends an already loaded mapblock to the player.
+    * Returns `false` if nothing was sent (note that this can also mean that
```

Comment:
Wouldn't it be better to return true if the client already has the block

Reply:
Yes, but I'm just describing the current state.